samples/connectedhomeip/lock/src/util/LEDWidget.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
```

Comment:
Do you think it would be good idea to move src/util content to some common directory for all samples, as soon we will create another one using the same files?

Reply:
Yes, I can do that, I guess.

Reply:
Done